scss/layout/_grid.scss

```diff
@@ -48,6 +50,21 @@
 
 // BREAKPOINT: LARGE
 @include media($lg) {
+  .grid--3-wide {
+    .grid__item {
+      @include span-columns(4);
+      margin-bottom: 0;
```

Comment:
Could the `margin-bottom` be included in a more generic rule? Doesn't seem like it would be specific to `.grid--3-wide` (or `.grid--4-wide`).